pkg/nsx/services/realizestate/realize_state.go

```diff
@@ -76,3 +75,33 @@ func (service *RealizeStateService) CheckRealizeState(backoff wait.Backoff, inte
 		return fmt.Errorf("%s not realized", intentPath)
 	})
 }
+
+func (service *RealizeStateService) GetPolicyTier1UplinkPortIP(intentPath string) (string, error) {
+	results, err := service.NSXClient.RealizedEntitiesClient.List(intentPath, nil)
+	err = nsxutil.TransNSXApiError(err)
+	if err != nil {
+		return "", err
+	}
+
+	for _, result := range results.Results {
+		extendAttributes := result.ExtendedAttributes
+		if len(extendAttributes) == 0 {
+			continue
+		}
+		for i := range extendAttributes {
+			if extendAttributes[i].Key != nil && *extendAttributes[i].Key == "IpAddresses" {
+				if len(result.IntentPaths) == 1 {
+					for _, ip := range extendAttributes[i].Values {
+						parts := strings.Split(ip, "/")
+						if len(parts) != 2 {
+							log.Info("Invalid tier1 uplink port IP found", "IP", ip, "Path", intentPath)
```

Comment:
log.Error?

Reply:
I removed this log since it's not important. Also, if we log as an error, we should return this error instead of continuation.